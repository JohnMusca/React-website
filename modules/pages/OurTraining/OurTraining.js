import React from 'react'
import Header from '../../common/Header/Header'
import Body from '../../common/Body/Body'
import Footer from '../../common/Footer/Footer'

//Content passed into body component.
var content="<p>Our training content</p>";

/**
* Our Training page component used to render the page.
*
* Props:      None
* 
* State vars: None
*/
class OurTraining  extends React.Component {
  render () {
    return (
      <div>  
      <div class="hero-image-training"></div>
        <Body heading="Our training" content={content}/>
        <Footer />
      </div>
    )
  }
}

export default OurTraining;
