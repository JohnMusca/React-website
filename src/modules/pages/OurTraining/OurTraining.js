import React from 'react'
import Header from '../../common/Header/Header'
import Body from '../../common/Body/Body'
import Footer from '../../common/Footer/Footer'


var instagram_link = "<span style='padding-left:10px;'><iframe src='//instagram.com/p/VID_ID/embed' width='230' height='300' frameborder='0' scrolling='no' allowtransperency='true'></iframe></span>";

//Content passed into body component.
var content="<p>Our Training comprises a variety and combination of:</p>" +
            "<ul>" +
              "<li>Practical instruction of Buk Sing philosophies and applications;</li>" + 
              "<li>Strong focus on combat techniques and tactics;</li>" + 
              "<li>Partner drills / Pad work;</li>" + 
              "<li>Sparring;</li>" + 
              "<li>Kung Fu Forms including use of weapons;</li>" + 
              "<li>Wooden dummy training.</li>" + 
            "</ul>" +
            "<p>Sifu Gary Mallia believes that effort equals reward and that the results from his instruction, will improve:</p>" + 
            "<ul>" + 
              "<li>Self Confidence and Self Esteem;</li>" + 
              "<li>Ability to comprehensively and effectively defend against a variety of opponents;</li>" + 
              "<li>Physical Fitness and Well Being;</li>" + 
              "<li>Flexibility, Balance and Co-Ordination.</li>" + 
            "</ul>" + 
            "<p>" + 
            instagram_link.replace('VID_ID', 'BZe5g8QjVyw') + 
            instagram_link.replace('VID_ID', 'BcUS3zMDONZ') + 
            instagram_link.replace('VID_ID', 'BZXa8wgjSr7') + 
            "</p>";

/**
* Our Training page component used to render the page.
*
* Props:      None
* 
* State vars: None
*/
class OurTraining  extends React.Component {

  constructor(props) {
    super(props);
  }
  
  render () {
    return (
      <div>  
      <div className="hero-image-training"></div>
        <Body heading="Our training" content={content}/>
        <Footer />
      </div>
    )
  }
}

export default OurTraining;
