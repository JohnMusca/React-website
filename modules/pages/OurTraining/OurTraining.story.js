import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import { BrowserRouter } from 'react-router-dom'
import OurTraining from './OurTraining'

storiesOf('Pages', module)
  .add('OurTraining', () => (
      <OurTraining />
  ));
