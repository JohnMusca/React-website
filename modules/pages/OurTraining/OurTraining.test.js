import React from 'react';
import OurTraining from './OurTraining';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom'

test('OurTraining component', () => {
  const component = renderer.create(
		<OurTraining />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
