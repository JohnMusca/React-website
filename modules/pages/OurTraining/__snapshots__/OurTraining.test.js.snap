// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurTraining component 1`] = `
<div>
  <div>
    <p>
      Our training content
    </p>
  </div>
  <div>
    <p>
      Â© 
      2018
       Bussiness Name
    </p>
  </div>
</div>
`;
